src/app.js

```diff
@@ -0,0 +1,64 @@
+import axios from 'axios';
+
+const searchForm = document.getElementById('search-form');
+searchForm.addEventListener('submit', searchCountry);
+
+const countryInfoBox = document.getElementById('search-result');
+const errorMessageBox = document.getElementById('error-message');
+
+function searchCountry(e) {
+    e.preventDefault();
+    const queryField = document.getElementById('query-field');
+    fetchCountryDetails(queryField.value);
+    queryField.value = '';
+}
+
+async function fetchCountryDetails(name) {
+    countryInfoBox.innerHTML = ``;
+    errorMessageBox.innerHTML = ``;
+
+    try {
+        const result = await axios.get(`https://restcountries.com/v2/name/${name}`);
+        const country = result.data[0];
```

Comment:
```suggestion
        const { name, flag, capital, subregion, population, currencies, languages } = result.data[0];
```
Dan hoef je niet overal `country.` vóór te zetten 😉